plugins/nanospell/plugin.js

```diff
@@ -267,6 +254,9 @@
 				}
 				return true;
 			});
+			editor.on(EVENT_NAMES.START_RENDER, render, self);
+			editor.on(EVENT_NAMES.START_SCAN_WORDS, scanWords, self);
+			editor.on(EVENT_NAMES.START_CHECK_WORDS, checkWords, self);
```

Comment:
curious why you moved these.  Not opposed.  I was floating between these two options for a while.